docs/docs/partials/_storage-template.md

```diff
@@ -17,7 +17,7 @@ Year/Year-Month-Day/Filename.Extension
 <img src={require('./img/storage-template.png').default} width="100%" title="Storage Template Setting" />
 
 :::tip
-By default, special characters will be converted to an HTML entity (for example, `&` -> `&amp;`). To prevent this, wrap the variable in an extra set of braces (for example, `{{{album}}}`). You can learn more about this [here](https://handlebarsjs.com/guide/expressions.html#html-escaping) and [here](https://github.com/immich-app/immich/issues/4917).
+The `{{album}}` variable will expand to the name of the album to which the file was most recently added to. By default, special characters will be converted to an HTML entity (for example, `&` -> `&amp;`). To prevent this, wrap the variable in an extra set of braces (for example, `{{{album}}}`). You can learn more about this [here](https://handlebarsjs.com/guide/expressions.html#html-escaping) and [here](https://github.com/immich-app/immich/issues/4917).
```

Comment:
I don't think it is "most recently added to".  It is actually based on when the album was created. We might want to change this, but that is what it looks like it does right now.

![image](https://github.com/immich-app/immich/assets/4334196/f94c7152-9622-4517-94ec-faba6cf18e06)